packages/file-upload/package.json

```diff
@@ -0,0 +1,146 @@
+{
+	"name": "@gov.au/file-upload",
+	"version": "0.1.0",
+	"description": "Upload file control",
+	"keywords": [
+		"auds",
+		"pancake",
+		"pancake-module",
+		"sass",
+		"scss",
+		"css",
+		"gov.au",
+		"Commonwealth of Australia",
+		"design guide",
+		"design system"
+	],
+	"scripts": {
+		"postinstall": "pancake",
+		"test:a11y": "node ../../scripts/a11y.js",
+		"test:helper": "node ../../scripts/helper.js test",
+		"test": "npm-run-all --parallel test:*",
+		"prepublish": "npm run test:helper && npm run build:pre",
+		"build:pre": "node ../../scripts/helper.js precompile publish",
+		"build:js": "npm run build:pre && node ../../scripts/helper.js compile",
+		"build:react": "cd tests/react/ && webpack",
+		"build": "npm run build:pre && npm run build:js && npm run build:react",
+		"serve": "browser-sync tests --files \"tests/**/*.html, tests/**/*.css, tests/**/*.js\"",
+		"watch:js": "onchange \"src/js/*.js\" -- npm run build:js",
+		"watch:jsx": "onchange \"src/js/react.js\" \"tests/react/index.js\" -- npm run build",
+		"watch:sass": "onchange \"src/sass/*.scss\" \"tests/site/test.scss\" -- npm run build",
+		"watch": "npm run build && npm-run-all --parallel serve watch:*"
+	},
+	"pancake": {
+		"auto-save": true,
+		"plugins": true,
+		"ignore": [],
+		"pancake-module": {
+			"version": "1.0.0",
+			"plugins": [
+				"@gov.au/pancake-sass",
+				"@gov.au/pancake-js",
+				"@gov.au/pancake-react",
+				"@gov.au/pancake-json"
+			],
+			"sass": {
+				"path": "lib/sass/_module.scss",
+				"sass-versioning": true
+			},
+			"js": {
+				"path": "lib/js/module.js"
+			},
+			"react": {
+				"path": "lib/js/react.js"
+			}
+		},
+		"json": {
+			"enable": false,
+			"location": "pancake/",
+			"name": "pancake",
+			"content": {
+				"name": true,
+				"version": true,
+				"dependencies": true,
+				"path": true,
+				"settings": true
+			}
+		},
+		"css": {
+			"minified": true,
+			"modules": false,
+			"browsers": [
+				"last 2 versions",
+				"ie 8",
+				"ie 9",
+				"ie 10"
+			],
+			"location": "pancake/css/",
+			"name": "pancake.min.css"
+		},
+		"sass": {
+			"modules": false,
+			"location": "pancake/sass/",
+			"name": "pancake.scss"
+		},
+		"react": {
+			"location": "pancake/react/"
+		}
+	},
+	"main": "lib/js/react.es5.js",
+	"dependencies": {
+		"@gov.au/buttons": "^3.0.6",
+		"@gov.au/core": "^3.0.0",
+		"@gov.au/pancake": "~1",
+		"@gov.au/pancake-js": "~1",
+		"@gov.au/pancake-json": "~1",
+		"@gov.au/pancake-react": "~1",
+		"@gov.au/pancake-sass": "~2"
+	},
+	"peerDependencies": {
+		"@gov.au/core": "^3.0.0",
+		"@gov.au/buttons": "^3.0.6"
+	},
+	"devDependencies": {
```

Comment:
We pushed through a pretty large change only yesterday where we significantly bumped up the dependencies we use across all packages / scripts. 

It's bad timing I'd say, can you please rebase/merge this branch with the latest `develop`